app/controllers/application_controller/report_data_additional_options.rb

```diff
@@ -24,6 +24,7 @@ class ApplicationController
     :supported_features_filter,
     :clickable,
     :no_checkboxes,
+    :no_checkboxes_clicked,
```

Comment:
For my knowledge what is the difference between no_checkboxes and no_checkboxes_clicked?  Asking because I generally avoid negative terms when naming, but no_checkboxes was there already.

I generally avoid negative terms when naming because it almost always leads to double and triple negatives.  (e.g. `if !no_checkboxes_clicked` as opposed to `if checkboxed_clicked`

Reply:
@Fryguy no_checkboxes is to display simple table without checkboxes for selection
@MelsHyrule Could you rename this variable?

Reply:
renaming to `checkboxes_clicked`